x-pack/plugins/security_solution/public/management/components/endpoint_responder/endpoint_response_actions_console_commands.tsx

```diff
@@ -38,6 +47,27 @@ const pidValidator = (argData: ParsedArgData): true | string => {
   }
 };
 
+const commandToCapabilitiesMap = new Map<ResponderCommands, ResponderCapabilities>([
+  ['isolate', 'isolation'],
+  ['release', 'isolation'],
+  ['kill-process', 'kill_process'],
+  ['suspend-process', 'suspend_process'],
+  ['processes', 'running_processes'],
+]);
```

Comment:
I feel like we have these list of commands in too many places. We should have just one place where they are defined.  Is there a way we ca centralize them and avoid having to define multiple lists. I think that the server side also has a list of response actions defined somewhere (Joey would know - I seem to remember him having it when I reviewed the API PR for the new response actions)

Reply:
The array that Joey wrote for response actions `x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.ts` is a little different because it's mapping the `response action` to the `feature keys`, whereas this map is for the `responder command names` to the `endpoint capability`. The strings are super similar but there are still some differences.